playbooks/adhoc/uninstall.yml

```diff
@@ -329,21 +328,16 @@
       dest=/etc/sysconfig/docker
       regexp='(ADD_REGISTRY|BLOCK_REGISTRY|INSECURE_REGISTRY)=.*'
 
-  - name: Detect Docker storage configuration
-    shell: vgs -o name | grep docker
-    register: docker_vg_name
-    failed_when: false
-    changed_when: false
-
-  - name: Wipe out Docker storage contents
-    command: vgremove -f {{ item }}
-    with_items: "{{ docker_vg_name.stdout_lines }}"
-    when: docker_vg_name.rc == 0
+  - name: Umount docker storage
+    shell: umount /var/lib/docker/volumes
```

Comment:
Why would only `volumes` be unmounted? Should the playbook attempt to unmount other folders?

Reply:
I'm sorry, that's my mistake. And yes, your guessing is correct. 
This unmount intends to fix https://github.com/openshift/openshift-ansible/issues/7922 In other words, without unmount, https://github.com/openshift/openshift-ansible/issues/7922 often happened.